doc/content/reference/components/network-operations-center.md

```diff
@@ -47,32 +51,124 @@ The following is a brief description of each column in the table:
+
+The end device overview consists of the following panels:
+
+**End devices active** - a bar chart that visualizes the number of devices that have connected to the network in the selected period, either in the last hour, week, or day.
+
+**End device statistics** - a table layout that displays the **End Device ID**, **Application ID**, **Last Seen**, and **Last Packet Error Rate** of all the devices on the network in the selected period.
+
+The following is a brief description of each column in the table:
+
+- **End Device ID**: displays IDs of all the end devices on the network.
+- **Application ID**: displays the application ID for each device.
+- **Last Seen**: displays when the device last uplinked.
+- **Last Packet Error Rate**: .
+
 ## Gateway Details Dashboard
 The Gateway Details dashboard visualizes detailed insight into gateway statistics in the selected time period.
 
 {{< figure src="../gateway-details-dashboard.png" alt="gateway details dashboard" >}}
 
 It consists of the following panels:
 
-**Uptime** - the gauge chart that visualizes the gateway uptime, i.e. the fraction of the time in which the gateway has been connected to the Network Server.
+**Uptime** - a gauge chart that visualizes the amount of time the gateway has been connected to the Network Server, represented as a percentage out of 100.
 
-{{< figure src="../uptime.png" alt="uptime" >}}
+{{< figure src="../gateway-uptime.png" alt="uptime" >}}
 
-**Lowest RSSI (LoRa)** - the graph that visualizes data about the lowest Received Signal Strength Indicator (RSSI) in dBm for each channel. 
+**Lowest RSSI (LoRa)** - a graph that visualizes data about the lowest Received Signal Strength Indicator (RSSI) in dBm for each channel. 
 
-{{< figure src="../rssi.png" alt="lowest rssi" >}}
+{{< figure src="../gateway-rssi.png" alt="lowest rssi" >}}
 
 **Traffic** - a time series graph that visualizes the uplink and downlink rates in packets per second.
 
-{{< figure src="../traffic.png" alt="traffic" >}}
+{{< figure src="../gateway-traffic.png" alt="traffic" >}}
 
 **Packets per channel** - a heat map that visualizes the number of uplink data packets for each channel.
 
-{{< figure src="../packets-per-channel.png" alt="packets per channel" >}}
+{{< figure src="../gateway-packets-per-channel.png" alt="packets per channel" >}}
 
-**Channel utilization** - a graph that visualizes the percentage of total utilization on each channel over a time interval.
+**Channel utilization** - a chart displaying the percentage of packets received on each channel.
 
-{{< figure src="../channel-utilization.png" alt="channel utilization" >}}
+{{< figure src="../gateway-channel-utilization.png" alt="channel utilization" >}}
 
 The Packets per channel and Channel utilization graphs can be used to find out which channels are overused and which channels are prone to collisions.
 
+**Packets per Data Rate** - a time series graph that visualizes the uplink and downlink rates in packets per second.
```

Comment:
I think this graph shows how many packets were sent over which data rate. How do you see which rate is uplink and which is downlink, and where do you see packets per second?

Reply:
@adriansmares

Reply:
This is strictly about uplinks, and it is represented in the number of uplinks observed in the bucket interval.

Reply:
Fixed